tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -2986,6 +2986,20 @@ namespace reverse_iterator_test {
         !std::sized_sentinel_for<reverse_iterator<simple_no_difference>, reverse_iterator<simple_no_difference>>);
 } // namespace reverse_iterator_test
 
+namespace lwg3420 {
+    // Validate that we can ask for the iterator_traits of a type with no * operator for which checking copyability

```

Comment:
Can/should this say `operator*` for increased searchability?

Reply:
Sure, that seems fine.
```suggestion
    // Validate that we can ask for the iterator_traits of a type with no operator* for which checking copyability
```